app/apollo/resolvers/channel.js

```diff
@@ -405,6 +405,9 @@ const channelResolvers = {
           }
         } ) );
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {name: newChannelObj.name, uuid: newChannelObj.uuid};
```

Comment:
pluginContext needs to include details about created version(s) and subscription(s), if created.